### PR TITLE
[Model Monitoring] Fix label columns in the monitoring serving graph and model endpoint object

### PR DESCRIPTION
**Pull request by Eyal-Danieli**
When creating a model endpoint, the feature names are taken from the feature stats that are stored under the model object. However, the feature stats also include the label columns. Therefore, the model endpoint stored the label columns in both  `model_endpoint.spec.feature_names` and `model_endpoint.spec.labels`. In this PR, we remove the label columns from `model_endpoint.spec.feature_names`.

In addition, the model monitoring serving graph (which defined in `model_monitoring/stream_processing_fs.py`) was referred to a wrong key when trying to process the outputs from the event dictionary. this PR also fix that issue (the correct key is `label_names` and not `label_columns`).

### CODE REVIEW

File: tests/system/model_monitoring/test_model_monitoring.py [hunk 1]
```diff
@@ -536,6 +536,12 @@ def test_model_monitoring_with_regression(self):
             == mlrun.api.schemas.ModelMonitoringMode.enabled.value
         )
 
+        # TODO uncomment the following assertion once the auto trainer function
```
Comment (guy1992l) [inline tests/system/model_monitoring/test_model_monitoring.py:539]:
```suggestion
        # TODO: uncomment the following assertion once the auto trainer function
```

**Review by guy1992l — COMMENTED** [review]
Add doc string to the function you added, although I don't understand why do you need this special data structure, where even holds another dict with the features values. Makes more sense to parse the event and get it instead of storing the values twice.

File: mlrun/model_monitoring/stream_processing_fs.py [hunk 2]
```diff
@@ -930,27 +930,44 @@ def do(self, event: typing.Dict):
...
+    @staticmethod
+    def _map_dictionary_values(
+        event: typing.Dict,
+        named_iters: typing.List,
+        values_iters: typing.List,
+        mapping_dictionary: str,
+    ):
+        """Adding name-value pairs to event dictionary based on two provided lists of names and values. These pairs
+        will be used mainly for the Parquet target file. In addition, this function creates a new mapping dictionary of
+        these pairs which will be unpacked in ProcessBeforeTSDB step"""
```
[the remark this answers is not in the document]
Comment (Eyal-Danieli) [reply mlrun/model_monitoring/stream_processing_fs.py:965]: Right, I will fix the docstring.
Regarding the data structure, not sure that were the historical reasons for creating another dictionary with features values and then unpacked this dictionary in the next steps (for the TSDB). In the current PR, I didn't change the logic of this part, just fixing the name of the label columns and put everything in a more elegant function. 
Anyway, I can fix this data structure but it's a different PR.

File: mlrun/model_monitoring/stream_processing_fs.py [hunk 3]
```diff
@@ -930,27 +930,44 @@ def do(self, event: typing.Dict):
...
+        event[mapping_dictionary] = {}
+        for name, value in zip(named_iters, values_iters):
+            event[name] = value
+            event[mapping_dictionary][name] = value
```
[the remark this answers is not in the document]
Comment (Eyal-Danieli) [reply mlrun/model_monitoring/stream_processing_fs.py:969]: See my previous answer

File: mlrun/model_monitoring/stream_processing_fs.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (guy1992l) [reply mlrun/model_monitoring/stream_processing_fs.py]: got it :+1:

**Review by guy1992l — APPROVED** [review]

File: mlrun/model_monitoring/stream_processing_fs.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (guy1992l) [reply mlrun/model_monitoring/stream_processing_fs.py]: but write yourself somewhere this needed to change someday :)

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**